sechub-developertools/scripts/sechub-api.sh

```diff
@@ -102,6 +181,34 @@ function sechub_project_scan_list {
 }
 
 
+function sechub_project_unassign_profile {
+  curl $CURL_PARAMS -i -X DELETE -H 'Content-Type: application/json' "$SECHUB_SERVER/api/admin/config/execution/profile/$2/project/$1" | $RESULT_FILTER | $JSON_FORMATTER
```

Comment:
Whenever you use two or more method arguments, they should be put into variables. The problem is, that functions in Shell scripts don't allow arguments in the signature. As a result, it is difficult to see the order of the variables. Please use variables properly named variables, to make the order clearly visible.

```
profile_id=$2
project_id=$1

curl $CURL_PARAMS …
```